Sources/ReplyService/Activity+ReplyRequest.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Activity+ReplyRequest.swift
+//  SwiftBot
+//
+//  Created by Andrew Tokarev on 27/06/2017.
+//
+//
+
+import Foundation
+import BotsKit
+
+extension Activity : ReplyRequest {
```

Comment:
What do you propose with that? 

Currently transformation is hidden in Protocol/Extension Implementation. But we can obviously build some other entity that does necessary transformations